### ISSUE TITLE
Cannot access from pages loaded over HTTPS

### ISSUE THREAD

**Issue by joelamos** [ripper234/Whatever-Origin#18]
whateverorigin.org uses obsolete encryption, so when accessing it from a page loaded over HTTPS, you get the error message: 

The page at <blah> was loaded over HTTPS, but requested an insecure script 'http://whateverorigin.org/get?url=...'. This request has been blocked; the content must be served over HTTPS.

**Comment by Foorack** [comment]
@joelamos Hellu.
I recommend using the following path if HTTPS is an requirement:
```https://whateverorigin.herokuapp.com/get?url=...```

**Comment by ericljiang** [comment]
Is https://whateverorigin.herokuapp.com not the same app as http://whateverorigin.org? The issue in #2 and #10 doesn't seem to be fixed when I use the former even though #22 seems to have fixed it in the latter.

**Comment by ripper234** [comment]
No, apperantly whateverorigin.org points to https://whatever-origin.herokuapp.com/, not https://whateverorigin.herokuapp.com/ - I probably have a double setup at Heroku for some reason.

**Comment by reynaldichernando** [comment]
HTTPS issue has been fixed!
You can access it via https://whateverorigin.org
